mapbox-gl/mapbox-gl-tests.ts

```diff
@@ -1,4 +1,3 @@
-// These examples are from Mapbox (https://www.mapbox.com/mapbox-gl-js/examples/) and therefore under the copyright of Mapbox Inc.
```

Comment:
Could you leave the URL in? `// These examples adapted from Mapbox's examples (https://www.mapbox.com/mapbox-gl-js/examples)`